### PR TITLE
add current-queries.sh to playbook

### PR DESCRIPTION
**Pull request by bloodearnest**

### CODE REVIEW

File: jobrunner/etc/opensafely/ssh-banner [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This the OpenSAFELY TPP backend.
```
Comment (madwort) [inline jobrunner/etc/opensafely/ssh-banner:1]: as this is a TPP-specific message, should the filename be TPP-specific?

Comment (bloodearnest) [reply jobrunner/etc/opensafely/ssh-banner:1]: Hmm, yeah not sure about this

File: jobrunner/playbook.md [hunk 2]
```diff
@@ -222,6 +222,24 @@ for debugging then omit this flag.
 The command is idempotent so you can always run it again later with the
 `--cleanup` flag.
 
+### Debugging slow queries
+
+The only way to gauge whether a DB job is stuck it to look at the docker logs
```
Comment (madwort) [inline jobrunner/playbook.md:227]:
```suggestion
The only way to gauge whether a DB job is stuck is to look at the docker logs
```

File: jobrunner/etc/opensafely/ssh-banner (hunk 1 above)
Comment (bloodearnest) [reply jobrunner/etc/opensafely/ssh-banner]: removed

**Review by madwort — APPROVED** [review]

**Pull request merged**